libavfilter/vf_dnn_detect.c

```diff
@@ -211,6 +211,30 @@ static int read_detect_label_file(AVFilterContext *context)
     return 0;
 }
 
+static int check_output_nb(DnnDetectContext *ctx, DNNBackendType backend_type, int output_nb)
+{
+    switch(backend_type) {
+    case DNN_TF:
+        if (output_nb != 4) {
+            av_log(ctx, AV_LOG_ERROR, "Dnn detect filter with tensorflow backend needs 4 outputs, \
```

Comment:
it is not tf backend needs 4 outputs, but that we currently only support models with 4 outputs.